x/cronos/keeper/evmhooks_test.go

```diff
@@ -159,6 +163,47 @@ func (suite *KeeperTestSuite) TestEvmHooks() {
 				suite.Require().Equal(1, len(rsp.SendToEthereums))
 			},
 		},
+		{
+			"failed ibc transfer, invalid denom",
```

Comment:
```suggestion
			"failed ibc transfer, non-IBC denom",
```
`invalid` is not self-explanatory here